scripts/test_import_operations.sh

```
@@ -0,0 +1,295 @@
+# Function to check if response is successful
+check_response() {
+    local response="$1"
+    local expected_status="$2"
+    
+    if echo "$response" | jq -e '.error' > /dev/null 2>&1; then
+        print_error "API call failed: $(echo "$response" | jq -r '.error')"
+        return 1
+    fi
+    
+    if [ -n "$expected_status" ]; then
+        local status=$(echo "$response" | jq -r '.statusCode // .status // "unknown"')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)